### DISCUSSION TITLE
Update of statistics: Comparison of vaccinated/vaccinated according to the representation in the population per week

### DISCUSSION THREAD

**Discussion by frantiseklorenc** [msusicky/ockovani-covid#646] [Q&A] [answered]
Hi, could you please update graph in statistics: Comparison of vaccinated/vaccinated according to the representation in the population per week? Is I think best of all graphs available but the last update was 24.10 so it's not current. This kind of statistics is nowhere else available or at least I don't know about it. Thanks, you're doing great work.

**Comment by jan-stanek** [comment, marked as answer]
Hi, we don't have newer data. UZIS promised that regularly updated datased should be available this week, but it seems to be delayed. As it will be published we will use it.

**Reply by jan-stanek** [reply]
@Frixtr done
